meshtastic/interfaces/ble/interface.py

```
@@ -951,8 +951,8 @@ def scan() -> list[BLEDevice]:
                 logger.warning("Device scan failed: %s", e, exc_info=True)
                 return []
             except (
-                Exception  # noqa: BLE001
-            ) as e:  # pragma: no cover - defensive last resort
+                Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)